src/docs/sphinx/dev_guide/tensor_dual.rst

```diff
@@ -0,0 +1,405 @@
+.. _header-n0:
```

Comment:
Can you make this match up with the page title?

Reply:
I don't know too much about .rst, what are you asking me to do here?

Reply:
These are sphinx labels that you can reference in other documents to here.  So if they don't match up with the page title its hard to remember.

https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets

Reply:
Okay! It looks like this is another downside to my .md -> .rst workflow

Reply:
@jamiebramwell did you resolve this in your recent commits?

Reply:
Yep!